storage/client/queries/queries.go

```diff
@@ -236,7 +236,7 @@ const (
 				chain.entities.address AS entity_address,
 				chain.nodes.id AS node_address,
 				chain.accounts.escrow_balance_active AS escrow,
-				chain.commissions.schedule AS commissions_schedule,
+				COALESCE(chain.commissions.schedule, '{}'::json) AS commissions_schedule,
```

Comment:
this json field gets scanned into a struct. v4 apparently scanned NULLs into a default-valued struct; v5 errors out in that case.